linuxPackages.evdi: 1.5.0.2 -> 1.5.1

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools. This update was made based on information from https://repology.org/metapackage/evdi/versions.



meta.description for linuxPackages.evdi is: '"Extensible Virtual Display Interface"'.

[Release on GitHub](https://github.com/DisplayLink/evdi/releases/tag/v1.5.1)


[Compare changes on GitHub](https://github.com/DisplayLink/evdi/compare/v1.5.0.2...v1.5.1)

<details>
<summary>
Checks done (click to expand)
</summary>

- built on NixOS

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- directory tree listing: https://gist.github.com/7e422e2989371ad3e616dc489cc48b8e
- du listing: https://gist.github.com/25b0a0709eaf6223f534907a1212d200

</details>
<details>
<summary>
Rebuild report (click to expand)
</summary>

59 total rebuild path(s)

20 package rebuild(s)

20 x86_64-linux rebuild(s)
20 i686-linux rebuild(s)
0 x86_64-darwin rebuild(s)
19 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
displaylink
linuxPackages-libre.evdi
linuxPackages.evdi
linuxPackages_4_14.evdi
linuxPackages_4_18.evdi
linuxPackages_4_19.evdi
linuxPackages_4_4.evdi
linuxPackages_4_9.evdi
linuxPackages_hardened.evdi
linuxPackages_hardkernel_4_14.evdi
linuxPackages_hardkernel_latest.evdi
linuxPackages_latest-libre.evdi
linuxPackages_latest.evdi
linuxPackages_latest_hardened.evdi
linuxPackages_latest_xen_dom0.evdi
linuxPackages_latest_xen_dom0_hardened.evdi
linuxPackages_testing_bcachefs.evdi
linuxPackages_testing_hardened.evdi
linuxPackages_xen_dom0.evdi
linuxPackages_xen_dom0_hardened.evdi

</details>

<details>
<summary>
Instructions to test this update (click to expand)
</summary>

Either download from Cachix:
```
nix-store -r /nix/store/q31vd7lk1wq7qkd5scdp5rvpm88ip240-evdi-1.5.1 \
  --option binary-caches 'https://cache.nixos.org/ https://r-ryantm.cachix.org/' \
  --option trusted-public-keys '
  r-ryantm.cachix.org-1:gkUbLkouDAyvBdpBX0JOdIiD2/DP1ldF3Z3Y6Gqcc4c=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(r-ryantm's Cachix cache is only trusted for this store-path realization.)

Or, build yourself:
```
nix-build -A linuxPackages.evdi https://github.com/r-ryantm/nixpkgs/archive/45fa7b6f3659ddc4eaa5f4057b1aa7c906a77510.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/q31vd7lk1wq7qkd5scdp5rvpm88ip240-evdi-1.5.1
ls -la /nix/store/q31vd7lk1wq7qkd5scdp5rvpm88ip240-evdi-1.5.1/bin
```


</details>
<br/>

r-ryantm committed:
linuxPackages.evdi: 1.5.0.2 -> 1.5.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/evdi/versions

GrahamcOfBorg commented:
<!--REQUEST_ID=5217d227-eb01-4ac9-9d6a-bf8399b45759-->
No attempt on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50754&attempt_id=939ad57e-ee3f-40e1-8ae2-0912d5d84ddf)

The following builds were skipped because they don't evaluate on aarch64-linux: linuxPackages.evdi

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=5217d227-eb01-4ac9-9d6a-bf8399b45759-->
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50754&attempt_id=0244b8a9-3709-4557-9e82-e7b1c5e37868)

Attempted: linuxPackages.evdi

<details><summary>Partial log (click to expand)</summary><p>

```
shrinking RPATHs of ELF executables and libraries in /nix/store/q31vd7lk1wq7qkd5scdp5rvpm88ip240-evdi-1.5.1
shrinking /nix/store/q31vd7lk1wq7qkd5scdp5rvpm88ip240-evdi-1.5.1/lib/libevdi.so
shrinking /nix/store/q31vd7lk1wq7qkd5scdp5rvpm88ip240-evdi-1.5.1/lib/modules/4.14.81/kernel/drivers/gpu/drm/evdi/evdi.ko
wrong ELF type
strip is /nix/store/vcc4svb8gy29g4pam2zja6llkbcwsyiq-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/q31vd7lk1wq7qkd5scdp5rvpm88ip240-evdi-1.5.1/lib
patching script interpreter paths in /nix/store/q31vd7lk1wq7qkd5scdp5rvpm88ip240-evdi-1.5.1
checking for references to /build in /nix/store/q31vd7lk1wq7qkd5scdp5rvpm88ip240-evdi-1.5.1...
wrong ELF type
/nix/store/q31vd7lk1wq7qkd5scdp5rvpm88ip240-evdi-1.5.1
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=5217d227-eb01-4ac9-9d6a-bf8399b45759-->
No attempt on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50754&attempt_id=6ee0c462-9af3-4099-8d16-3540a3dedbcf)

The following builds were skipped because they don't evaluate on x86_64-darwin: linuxPackages.evdi

<details><summary>Partial log (click to expand)</summary><p>

```
while evaluating the attribute 'linux_4_14' at /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/top-level/all-packages.nix:14357:3:
while evaluating 'callPackageWith' at /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/lib/customisation.nix:108:35, called from /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/top-level/all-packages.nix:14357:16:
while evaluating 'makeOverridable' at /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/lib/customisation.nix:67:24, called from /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/lib/customisation.nix:112:8:
while evaluating anonymous function at /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/os-specific/linux/kernel/linux-4.14.nix:1:1, called from /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/lib/customisation.nix:69:12:
while evaluating 'buildLinux' at /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/top-level/all-packages.nix:14655:16, called from /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/os-specific/linux/kernel/linux-4.14.nix:5:1:
while evaluating 'callPackageWith' at /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/lib/customisation.nix:108:35, called from /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/top-level/all-packages.nix:14655:23:
while evaluating 'makeOverridable' at /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/lib/customisation.nix:67:24, called from /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/lib/customisation.nix:112:8:
while evaluating anonymous function at /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/os-specific/linux/kernel/generic.nix:1:1, called from /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/lib/customisation.nix:69:12:
assertion failed at /private/var/lib/ofborg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/os-specific/linux/kernel/generic.nix:51:1

```
</p></details>

c0bw3b committed:
evdi: refresh meta

Licenses : lib is LGPL-2.1-only while module is GPL-2.0-only

Supported kernel versions: builds with 4.15 / 4.16 / 4.17 since commit 90c26648acbe9a2f7ce4e0331e39045819323bf2

c0bw3b commented:
I changed `meta.broken` because it should build with kernel 4.15 to 4.17 after:
https://github.com/DisplayLink/evdi/commit/90c26648acbe9a2f7ce4e0331e39045819323bf2

c0bw3b commented:
@GrahamcOfBorg build linuxPackages.evdi

GrahamcOfBorg commented:
<!--REQUEST_ID=2fe2ab20-1c40-431c-8d70-2a9ab58c4b09-->
No attempt on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50754&attempt_id=fd9d9019-0c66-49e4-a2fb-5a06cdecc4fd)

The following builds were skipped because they don't evaluate on aarch64-linux: linuxPackages.evdi

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=2fe2ab20-1c40-431c-8d70-2a9ab58c4b09-->
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50754&attempt_id=e4f47a16-0948-4fcd-9808-b4540c19293c)

Attempted: linuxPackages.evdi

<details><summary>Partial log (click to expand)</summary><p>

```
shrinking RPATHs of ELF executables and libraries in /nix/store/71w11b5ig5223yfs1v1yffjc4vccf9wg-evdi-1.5.1
shrinking /nix/store/71w11b5ig5223yfs1v1yffjc4vccf9wg-evdi-1.5.1/lib/libevdi.so
shrinking /nix/store/71w11b5ig5223yfs1v1yffjc4vccf9wg-evdi-1.5.1/lib/modules/4.14.82/kernel/drivers/gpu/drm/evdi/evdi.ko
wrong ELF type
strip is /nix/store/rpbg8gmqxhz8g61p1plz5d2srs84pvmv-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/71w11b5ig5223yfs1v1yffjc4vccf9wg-evdi-1.5.1/lib
patching script interpreter paths in /nix/store/71w11b5ig5223yfs1v1yffjc4vccf9wg-evdi-1.5.1
checking for references to /build in /nix/store/71w11b5ig5223yfs1v1yffjc4vccf9wg-evdi-1.5.1...
wrong ELF type
/nix/store/71w11b5ig5223yfs1v1yffjc4vccf9wg-evdi-1.5.1
```
</p></details>

c0bw3b commented:
@GrahamcOfBorg build displaylink

GrahamcOfBorg commented:
<!--REQUEST_ID=ed482d79-397e-4966-bcbe-3b64ad1aafbc-->
No attempt on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50754&attempt_id=8b0e501b-877a-4230-ab1e-de3714811776)

The following builds were skipped because they don't evaluate on aarch64-linux: displaylink

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowUnfree = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowUnfree = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=ed482d79-397e-4966-bcbe-3b64ad1aafbc-->
No attempt on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50754&attempt_id=5f23b228-a194-4001-bb9e-19a62f97b054)

The following builds were skipped because they don't evaluate on x86_64-linux: displaylink

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowUnfree = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowUnfree = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>
